ci/docker/Dockerfile.build.ubuntu_nightly_gpu

```diff
@@ -29,6 +29,7 @@ COPY install/deb_ubuntu_ccache.sh /work/
 RUN /work/deb_ubuntu_ccache.sh
 
 COPY install/ubuntu_python.sh /work/
+COPY install/requirements /work/
```

Comment:
so the test environment is in the '/work/` directory?

Reply:
yes, work directory is created inside the docker and all required files are copied there.